.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,106 @@
+name: Bug
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues. If this is regarding NS functionality, please report to NS customer Support Serivce.
+        
+        ---
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: dropdown
+    id: tool
+    attributes:
+      label: Select the SuiteCloud tool where you encountered the issue
+      options:
+        - SuiteCloud CLI
+        - SuiteCloud Extension for VSCode
+        - SuiteCloud Unit Testing
+    validations:
+      required: true
+  - type: input
+    id: toolVersion
+    attributes:
+      label: Version of the SuiteCloud Tool
+      description: |
+                    For CLI run: `suitecloud --version`
+
+                    For VSCode Extension: `ctrl+shift+x` , search for suitecloud extension, Check the Version in front of Extension Name (Not in the description below)
+
+                    For Unit-Testing package: Open package.json from your suitecloud project, check the version on "@oracle/suitecloud-unit-testing": ^1.2.1"
+      placeholder: "1.2.0"
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: Operating system
+      description: "Which operating system do you use? Please provide the version as well."
+      placeholder: "macOS Big Sur 11.5.2"
+    validations:
+      required: true  
+  - type: input
+    id: Node
+    attributes:
+      label: Node Version/ VSCode Version
```

Comment:
Can you please add a blank space character after node version ?  `label: Node Version / VSCode Version`

Reply:
Agree